### PR TITLE
added g2psandbox superset chart

### PR DESCRIPTION
**Pull request by somanath21**
Created g2psandbox superset chart for fineract

### CODE REVIEW

File: helm/fineract/values.yaml [hunk 1]
```diff
@@ -17,8 +17,7 @@ fineract:
     path: "/"
     wildcardhostname: "*.sandbox.fynarfin.io"
     annotations:
-      kubernetes.io/ingress.class: "nginx"
-      nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
```
Comment (fynmanoj) [inline helm/fineract/values.yaml:21]: we dont need to delete this

File: helm/g2pconnect-superset/values.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
...TO tenant_server_connections (id, schema_server, schema_name, schema_server_port, schema_username, schema_password, auto_update, pool_initial_size, pool_validation_interval, pool_remove_abandoned, pool_remove_abandoned_timeout, pool_log_abandoned, pool_abandon_when_percentage_full, pool_test_on_borrow, pool_max_active, pool_min_idle, pool_max_idle, pool_suspect_timeout, pool_time_between_eviction_runs_millis, pool_min_evictable_idle_time_millis, schema_connection_parameters, readonly_schema_server, readonly_schema_name, readonly_schema_server_port, readonly_schema_username, readonly_schema_password, readonly_schema_connection_parameters) VALUES (2, \"fineract-mysql\", \"gorilla\", 3306, \"mifos\", \"password\", 1, 5, 30000, 1, 60, 1, 50, 1, 40, 20, 10, 60, 34000, 60000, \"serverTimezone=UTC&useLegacyDatetimeCode=false&sessionVariables=time_zone=\\\"-00:00\\\"\", NULL, NULL, NULL, NULL, NULL, NULL)" && mysql -h fineract-mysql -u root -p4ET6ywqlGt fineract_tenants -e "INSERT INTO tenants (id, identifier, name, timezone_id, country_id, joined_date, created_date, lastmodified_date, oltp_id, report_id) VALUES (2, \"gorilla\", \"test2\", \"Asia/Kolkata\", NULL, NULL, NULL, NULL, 2, 2)";'
```
Comment (fynmanoj) [inline helm/g2pconnect-superset/values.yaml:12]: INSERT IGNORE is missing here: `"INSERT INTO tenants (id, identifier, name, timezone_id, country_id, joined_date, created_date, lastmodified_date, oltp_id, report_id) VALUES (2, \"gorilla\", \"test2\", \"Asia/Kolkata\", NULL, NULL, NULL, NULL, 2, 2)";'`

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: helm/fineract/values.yaml (hunk 1 above)
Comment (somanath21) [reply helm/fineract/values.yaml:21]: I have re-added it in next commit

Comment (somanath21) [reply helm/fineract/values.yaml:21]: added in new commit

File: helm/g2pconnect-superset/values.yaml (hunk 2 above)
Comment (fynmanoj) [reply helm/g2pconnect-superset/values.yaml:12]: can you update this?

Comment (somanath21) [reply helm/g2pconnect-superset/values.yaml:12]: Updated in New commit

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**